sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -3,33 +3,35 @@
 
 package com.azure.cosmos;
 
+import io.netty.channel.ChannelOption;
+
 import java.time.Duration;
 
 /**
  * Represents the connection config with {@link ConnectionMode#DIRECT} associated with Cosmos Client in the Azure Cosmos DB database service.
+ * For performance tips on how to optimize Direct connection configuration,
+ * refer to performance tips guide: https://docs.microsoft.com/en-us/azure/cosmos-db/performance-tips-java-sdk-v4-sql?tabs=api-async
  */
 public final class DirectConnectionConfig {
     //  Constants
-    private static final Duration DEFAULT_IDLE_CHANNEL_TIMEOUT = Duration.ZERO;
     private static final Duration DEFAULT_IDLE_ENDPOINT_TIMEOUT = Duration.ofSeconds(70L);
-    private static final int DEFAULT_MAX_CHANNELS_PER_ENDPOINT = 30;
+    private static final int DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT = 30;
     private static final int DEFAULT_MAX_REQUESTS_PER_ENDPOINT = 10;
```

Comment:
DEFAULT_MAX_REQUESTS_PER_ENDPOINT should be DEFAULT_MAX_REQUESTS_PER_CONNECTION

Reply:
Good catch, thanks.